src/components/menu/__internal__/submenu/submenu.spec.js

```diff
@@ -773,4 +774,35 @@ describe("Submenu component", () => {
       expect(wrapper.find(MenuItem).length).toEqual(4);
     });
   });
+  describe("when it has Search as a child", () => {
+    const renderWithSearch = (openSubmenu, menuType, props) => {
+      return mount(
+        <MenuContext.Provider value={menuContextValues(openSubmenu, menuType)}>
+          <Submenu title="title" tabIndex={-1} {...props}>
+            <MenuItem>Apple</MenuItem>
+            <MenuItem>Banana</MenuItem>
+            <MenuItem variant="alternate">
+              <Search
+                defaultValue=""
+                placeholder="Dark variant"
+                variant="dark"
+                onChange={() => {}}
+              />
+            </MenuItem>
+          </Submenu>
+        </MenuContext.Provider>,
+        { attachTo: container }
+      );
+    };
+
+    it("should render with correct styles for search icon", () => {
+      wrapper = renderWithSearch(true, "dark");
+      assertStyleMatch(
+        {
+          color: "rgba(0,0,0,0.65)",
```

Comment:
Rather than hardcode a value that could be changed in the theme config, could you import the theme and reference `theme.menu.dark.searchIcon`?